src/sentry/models/groupsearchview.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.db.models.fields.hybrid_cloud_foreign_key import HybridCloudForeignKey
+from sentry.models.savedsearch import SortOptions
+
+
+@region_silo_model
+class GroupSearchView(DefaultFieldsModel):
+    """
+    A model for a user's view of the issue stream
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    name = models.TextField(max_length=128)
+    user_id = HybridCloudForeignKey("sentry.User", on_delete="CASCADE")
+    organization = FlexibleForeignKey("sentry.Organization")
+
+    query = models.TextField()
+    query_sort = models.CharField(
+        max_length=16, default=SortOptions.DATE, choices=SortOptions.as_choices(), null=True
```

Comment:
Will this ever be null?

Reply:
Yes - an empty search query for the view is a valid query

Reply:
Since you have a default here, when will this sort ever be null though?

Reply:
🤦 My bad I read the code wrong and thought the null was applying to the query, not the query_sort. You're right - since there is a default sort, `query_sort` should never be null. Fixing that now.